xchange-binance/src/main/java/org/knowm/xchange/binance/dto/account/BinanceFlexiblePosition.java

```diff
@@ -0,0 +1,83 @@
+package org.knowm.xchange.binance.dto.account;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.math.BigDecimal;
+import java.util.Map;
+import lombok.Getter;
+
+@Getter
+public final class BinanceFlexiblePosition {
+
+  @JsonProperty("totalAmount")
+  private final BigDecimal totalAmount;
+
+  @JsonProperty("tierAnnualPercentageRate")
+  private final Map<String, BigDecimal> tierAnnualPercentageRate;
+
+  @JsonProperty("latestAnnualPercentageRate")
+  private final BigDecimal latestAnnualPercentageRate;
+
+  @JsonProperty("yesterdayAirdropPercentageRate")
+  private final BigDecimal yesterdayAirdropPercentageRate;
+
+  @JsonProperty("asset")
+  private final String asset;
+
+  @JsonProperty("airDropAsset")
+  private final String airDropAsset;
+
+  @JsonProperty("canRedeem")
+  private final Boolean canRedeem;
+
+  @JsonProperty("collateralAmount")
+  private final BigDecimal collateralAmount;
+
+  @JsonProperty("productId")
+  private final String productId;
+
+  @JsonProperty("yesterdayRealTimeRewards")
+  private final BigDecimal yesterdayRealTimeRewards;
+
+  @JsonProperty("cumulativeBonusRewards")
+  private final BigDecimal cumulativeBonusRewards;
+
+  @JsonProperty("cumulativeRealTimeRewards")
+  private final BigDecimal cumulativeRealTimeRewards;
+
+  @JsonProperty("cumulativeTotalRewards")
+  private final BigDecimal cumulativeTotalRewards;
+
+  @JsonProperty("autoSubscribe")
+  private final Boolean autoSubscribe;
+
+  public BinanceFlexiblePosition(
```

Comment:
can be omitted here and in other dtos

Reply:
Yes, replaced with @Builder + @Jacksonized